.azuredevops/appcenter-sdk-unity-build-release-1es.yml

```diff
@@ -4,6 +4,19 @@ trigger:
 pr:
 - master
 - develop
+
+variables: 
+- name: EDITOR_URL_WIN
+  value: 'https://download.unity3d.com/download_unity/61fce66342ad/Windows64EditorInstaller/UnitySetup64-2018.4.18f1.exe'
+- name: IL2CPP_SUPPORT_URL
```

Comment:
Where is this variable used?

Reply:
It used in build.cake script, that running in Install Unity task. 
https://github.com/microsoft/appcenter-sdk-unity/blob/b85e7669d71a38c304646038ad26bc3819ca10c5/build.cake#L378